src/gui/tray/Window.qml

```diff
@@ -851,7 +851,55 @@ ApplicationWindow {
             anchors.right: trayWindowMainItem.right
         }
 
+        Loader {
+            id: newActivitiesButtonLoader
+            anchors.top: activityList.top
+            anchors.topMargin: 5
+            width: 150
+            height: 40

```

Comment:
This should be a set value in `Style` or use values already in `Style` to keep things consistent